src/compat_wrappers.h

```diff
@@ -8,6 +8,10 @@
 
 #include "b-em.h"
 
+#ifndef WIN32
+#define HAVE_STPCPY
+#endif
```

Comment:
Ah, I hadn't appreciated this was already in autotools. Is is there by default, or did it just get recently added? I'll invert the logic, as suggested by Steve.